### DISCUSSION TITLE
My The hill Turkish Translate Project

### DISCUSSION THREAD

**Discussion by Burkicannjr** [rocksdanister/lively#2298] [Show and tell]
Hello I'm Burkicannjr and I'm new here, but I definitely enjoy helping some software developers. And that's why I wanted to translate The Hill wallpaper into Turkish. I can share the file it if you want. Edit: There were no Turkish characters in the Dancing Font, so I replaced the Dancing font with a version with Turkish characters, everything is the same, only Turkish characters are present.
![Ekran görüntüsü 2024-06-29 185412](https://github.com/rocksdanister/lively/assets/174200883/c3c81e3c-8c01-489c-a1bc-c7d9966f2aca)
![Ekran görüntüsü 2024-06-29 185345](https://github.com/rocksdanister/lively/assets/174200883/3ec8fb33-5bc7-41c9-b9d8-45c152bb00e5)
![Ekran görüntüsü 2024-06-29 185457](https://github.com/rocksdanister/lively/assets/174200883/58a719a3-4323-4fe6-a810-f956cfd5113a)

**Comment by rocksdanister** [comment]
That's good idea. I wonder if its possible combine i18n localization library and multiple fonts into a single package or maybe find a font that works with most languages🤔

**Reply by Burkicannjr** [reply]
Thank you very much for reviewing my project. Frankly, I don't know what you mean by i18n. By the way, I'm not someone who understands software. I just like translation and browsing through folders in programs. If you have any problems, I can try to help. Normally, I never had a project like this in mind, but I am a young boy who has been interested in translation for the last 3-4 months. Just a few months ago, when I translated tme in Rainmeter to Turkish, I started to become curious and I wanted to help someone, so I shared it saying that I could at least support a software that I enjoy using. By the way, if you ask me why I told you about this, frankly, I wrote it down to at least write it down somewhere and keep it there instead of keeping it inside me.
